openslides/core/static/js/core/base.js

```diff
@@ -52,6 +52,13 @@ angular.module('OpenSlidesApp.core', [
         var recInterval = null;
         $rootScope.connected = false;
 
+        var websocketProtocol;
+        if (location.protocol == 'https:') {
+            websocketProtocol = 'wss://';
```

Comment:
The double slash is not part of the protocol. Please remove it from this variable and re-add it in line 83.